Euclidean_Space.R

```diff
@@ -0,0 +1,83 @@
+#Euclidean_Space
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+EuclideanSpace <- setRefClass("EuclideanSpace",
+                              contains = Manifold,
+                              methods = list(
+
+                                Init <- function(self, dimension){
+                                  stopifnot(dimension %% 1 == 0, dimension > 0)
+                                  self$dimension <- dimension
+                                  # self$metric <-
+                                },
+
+                                Belongs <- function(self,point){
+                                  # Evaluate if a point belongs to the Euclidean space.
+                                  point <- gs$backend$to_ndarray(point, to_ndim = 2)
+                                  n_points = point$shape

```

Comment:
Find a R function that outputs the shape of an array. Since `point` will likely not be a numpy array in R, it's likely that `point` will not have a `shape` attribute.